app/Http/routes.php

```diff
@@ -179,6 +179,10 @@
     Route::resource('volunteer','VolunteerController', ['only' => [
         'show', 'edit', 'update'
     ]]);
+     /**
+     * Volunteer view his events.
+     */
+    Route::get('volunteer/events','VolunteerController@showEvents');
```

Comment:
Since it will be part of the dashboard. We may make the route **dashboard/events**